app-frontend/src/app/components/colorCorrectPane/colorCorrectPane.module.js

```diff
@@ -0,0 +1,12 @@
+import angular from 'angular';
+import slider from 'angularjs-slider';
+
+import ColorCorrectPane from './colorCorrectPane.component.js';
+import ColorCorrectController from './colorCorrectPane.controller.js';
+
+const ColorCorrectModule = angular.module('components.colorCorrectPane', [slider]);
+
+ColorCorrectModule.component('colorCorrectPane', ColorCorrectPane);
```

Comment:
I've been prefixing components we make with "rf" IE "rfSceneItem" -> "`<rf-scene-item>`".